docs/Z-Score-normalization-script.md

```diff
@@ -5,22 +5,32 @@ For some data types, when uploading to cBioPortal, it is currently necessary to
 Please keep in mind that the z-scores are calculated using only patient data. Hence, overexpressed in this case implies higher expression than the average patient. Also, the source data on which the z-score data is based does not necessarily follow the normal distribution. If your data does not follow the normal distribution, the z-score threshold is less reliable and will result in more false positives or false negatives. You can consider log transforming your value before calculating z-scores to improve this. However, even the logged values may not follow the normal distribution, especially if the data is bimodal.
```

Comment:
Sorry I didn't catch this the first time, but in OncoPrint we have shifted to using "high" and "low" rather than "over/under-expressed" (see: http://bit.ly/2wnevrA). It might be good to use that same terminology in this document.

Reply:
Thanks @tmazor. Updated the introduction to use high/low.